frontend/src/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import ExploreHeader from './components/ExploreHeader.js'
+
+// this is optional really, if we import components directly React will be 
+// in the bundle, so we won't need a global reference
+window.React = React
+window.ReactDOM = ReactDOM
+
+window.Metamaps = window.Metamaps || {}
+window.Metamaps.ReactComponents = {
+  ExploreHeader
+}
```

Comment:
I envision either having a bunch of components available for use in the javascript files over in app/assets/javascripts, or this file could call ReactDOM.render a bunch of times.